### DISCUSSION TITLE
Korau: can I this lib in my KMM project

### DISCUSSION THREAD

**Discussion by a-blekot** [korlibs/korge#929] [Q&A]
Hi guys! @soywiz 
I have [simple project](https://github.com/a-blekot/memorize_shloka) with audio player functionality. Currently, I'm using ExoPlayer on the Android side and AVQueuePlayer in IOs (in another similar project).
I am wondering if I can use Korau for this purpose.

I need to cover two cases or at least one))
1) Play .mp3 files from **app assets** (I use MOKO-resources to store assets in the shared KMM module)

another app
2) Play .mp3 files from **URL or local file** (previously downloaded)

**Comment by soywiz** [comment]
Hey @a-blekot in theory it is possible. Not sure how Moko resources work, but in general if the asset is in the app assets folder, you can use the `resourcesVfs["file.mp3"].readSound()`. The best thing you can do is to try it for yourself and see if it works for your case. If it doesn't work please, let us know so we can try to fix/support your use-case

**Reply by a-blekot** [reply]
Hi! Thank you for your reply) I have read it only now =)

Ok, I will try this method!
